facebook/qsf-tools/generate-changelog.R

```diff
@@ -46,7 +48,7 @@ WAVE_COMPARE_MAP <- list(
 
 DIFF_COLS <- c(
   "question",
-  "matrix_subquestion",
+  "subquestion",
```

Comment:
CMU documentation still says `matrix_subquestion`, so the description is still valid. For UMD, the `subquestion` field is for both matrix and dropdown subquestion/component text.

Wichada has reviewed the codebooks.

Reply:
Not sure I understand your first sentence. Are you saying CMU microdata files will still contain a `matrix_subquestion` column?

Reply:
CMU codebook files still currently contain a `matrix_subquestion` column, yes. CMU doesn't have dropdowns that have extra component text, so I didn't regenerate the codebook files. Should I?

Reply:
If we don't plan to regenerate the CMU codebooks again, that's fine, we can leave the documentation as-is. I was just confused because I thought this meant the column name was changing in our files.

You may want to make sure Zheng changes the UMD microdata documentation, since a lot of the parts were copied from our documentation.